components/layouts/AlertDisplayer.tsx

```diff
@@ -92,6 +92,9 @@ const ALERTS: DictAlerts = {
     'L’identifiant Pôle emploi a bien été mis à jour',
   ajoutCommentaireAction:
     'Votre jeune a été alerté que vous avez écrit un commentaire',
+  qualificationSNP:
+    'L’action a été qualifiée en Situation Non Professionnelle et les informations ont été envoyées à i-milo',
```

Comment:
```suggestion
    'L’action a été traitée et qualifiée en Situation Non Professionnelle.'
```
   "Les informations ont été envoyées à i-milo" est en paragraphe, pas en titre.
cf. https://www.figma.com/file/ifExm8ybgmIqFmuECi2mwQ/Cr%C3%A9ation-d%E2%80%99une-SNP?node-id=673%3A12636